CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [next]
 
+- fix(IText): **NO** clear context top during rendering #8560
```

Comment:
let's be more clear on what happened here.
let's try to summarize what we changed with what because we are still clearing, just in a different moment of renderAll

Reply:
done